### ISSUE TITLE
Low 10-minute APGAR score

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5882]
**Preferred term label:**  Low 10-minute APGAR score

**Synonyms**  Low ten-minute APGAR score

**Definition (free text, please give PubMed ID)** PMID: 29262097

**Parent term (use hpo.jax.org/app)** Low APGAR score HP:0030917

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** 10-minute APGAR score of 0

**Synonyms** Ten-minute APGAR score of zero

**Definition (free text, please give PubMed ID)** PMID: 29262097

**Parent term (use hpo.jax.org/app)** Low 10-minute APGAR score

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** 10-minute APGAR score of 1

**Synonyms** Ten-minute APGAR score of one

**Definition (free text, please give PubMed ID)** PMID: 29262097

**Parent term (use hpo.jax.org/app)** Low 10-minute APGAR score

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** 10-minute APGAR score of 2

**Synonyms** Ten-minute APGAR score of two

**Definition (free text, please give PubMed ID)** PMID: 29262097

**Parent term (use hpo.jax.org/app)** Low 10-minute APGAR score

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** 10-minute APGAR score of 3

**Synonyms** Ten-minute APGAR score of three

**Definition (free text, please give PubMed ID)** PMID: 29262097

**Parent term (use hpo.jax.org/app)** Low 10-minute APGAR score

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** 10-minute APGAR score of 4

**Synonyms** Ten-minute APGAR score of four

**Definition (free text, please give PubMed ID)** PMID: 29262097

**Parent term (use hpo.jax.org/app)** Low 10-minute APGAR score

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** 10-minute APGAR score of 5

**Synonyms** Ten-minute APGAR score of five

**Definition (free text, please give PubMed ID)** PMID: 29262097

**Parent term (use hpo.jax.org/app)** Low 10-minute APGAR score

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** 10-minute APGAR score of 6

**Synonyms** Ten-minute APGAR score of six

**Definition (free text, please give PubMed ID)** PMID: 29262097

**Parent term (use hpo.jax.org/app)** Low 10-minute APGAR score

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added, thanks
